unstructured_inference/inference/layout.py

```diff
@@ -30,18 +31,30 @@ def __str__(self):
     def to_dict(self):
         return self.__dict__
 
+    def get_width(self):
+        # i.e: y2-y1
```

Comment:
On this and the other comments, update to `# NOTE(benjamin): i.e.: y2-y1` so we know who to refer questions to in the future.